ansible/psi/README.md

```diff
@@ -0,0 +1,36 @@
+Resources for setting up CI in PSI OpenStack
+============================================
+
+We have a "front-door-ci" project on the PSI OpenStack clouds rhos-d and rhos-01. The latter's nested KVM support and performance is good enough for our tasks runners.
+
+Access
+------
+Sign into the OpenShift web console on https://api.rhos-01.prod.psi.rdu2.redhat.com with group "redhat.com", your RedHat user name, and your Kerberos password. You need to be in the [front-door-ci group](https://rover.redhat.com/groups/group/front-door-ci) for this.
+
+The only change that was, and currently needs to be done manually is to add SSH and ICMP to the "default" security group. This step may be automated in the future.
+
+Automation setup
+----------------
+
+You need to install the OpenStack SDK, either with `sudo dnf install python3-openstacksdk` (on Fedora), or `pip install openstacksdk`.
```

Comment:
I also need `python3-openstackclient`

Reply:
I don't have that installed. It's not necessary for running the playbooks, you just need it if you want to run the `openstack` CLI command directly. I'll instead drop the `openstack` mention in README, it's redundant anyway.